### PR TITLE
Target .NET 8

### PR DESCRIPTION
**Pull request by RussKie**
Related https://github.com/gitextensions/gitextensions/pull/11240

### CODE REVIEW

**Comment by RussKie** [comment]
NB: this depends on https://github.com/gitextensions/gitextensions.extensibility/pull/30.

File: src/GitExtensions.PluginManager/GitExtensions.PluginManager.csproj [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
   <PropertyGroup>
     <UseWindowsForms>true</UseWindowsForms>
+    <UseWPF>true</UseWPF>
```
Comment (maraf) [inline src/GitExtensions.PluginManager/GitExtensions.PluginManager.csproj:5]: Why do we need this change?

**Comment by RussKie** [comment]
It fauls to compile without it

**Comment by RussKie** [comment]
> Tested locally, the `GitExtensions-v4.3.0.17614` won't offer the plugin in Plugin menu

Could you please describe the local testing procedure?

**Comment by maraf** [comment]
> Could you please describe the local testing procedure?

1. Publish `PackageManager.UI` 
2. Run `GitExtensions.PluginManager`

**Comment by RussKie** [comment]
@maraf do you think you could give me a hand in getting this over the finish line?

**Comment by maraf** [comment]
I'm happy to, but I'm AFK till 7/7. If you're willing to wait, I'm happy to jump on it.

**Comment by RussKie** [comment]
A gentle nudge

**Comment by maraf** [comment]
Are we trying to target latest GE on AppVeyor (GitExtensions-v4.3.0.17863) with this?

**Comment by maraf** [comment]
The same status as in https://github.com/gitextensions/gitextensions.pluginmanager/pull/73#issuecomment-1827752835. How can I debug MEF?

**Comment by maraf** [comment]
It should be ready to go! In review you can omit the last commit 61ebd489ec677682795e65c1badf890cc6e66b7b, it something VS suggested to me

**Comment by RussKie** [comment]
> It should be ready to go! In review you can omit the last commit [61ebd48](https://github.com/gitextensions/gitextensions.pluginmanager/commit/61ebd489ec677682795e65c1badf890cc6e66b7b), it something VS suggested to me

Thank you.
Do you think we could exclude [61ebd48](https://github.com/gitextensions/gitextensions.pluginmanager/commit/61ebd489ec677682795e65c1badf890cc6e66b7b) from this PR and push it separately?

File: src/GitExtensions.PluginManager/GitExtensions.PluginManager.csproj [hunk 2]
```diff
@@ -27,15 +28,12 @@
...
 
   <ItemGroup>
-    <Reference Include="GitUI">
-      <HintPath>$(GitExtensionsPath)\GitUI.dll</HintPath>
+    <Reference Include="GitExtensions.Extensibility">
+      <HintPath>$(GitExtensionsPath)\GitExtensions.Extensibility.dll</HintPath>
     </Reference>
     <Reference Include="GitUIPluginInterfaces">
       <HintPath>$(GitExtensionsPath)\GitUIPluginInterfaces.dll</HintPath>
     </Reference>
```
Comment (RussKie) [inline src/GitExtensions.PluginManager/GitExtensions.PluginManager.csproj:36]: Do we still need to reference this one?

**Comment by RussKie** [comment]
And, if you're think, it's ready to go, please approve, so I can I have it merged.

**Review by maraf — APPROVED** [review]

**Comment by RussKie** [comment]
Thank you!

**Pull request merged**